tests/core/test_simulation_builder.py

```diff
@@ -45,18 +46,20 @@ def __init__(self):
 @pytest.fixture
 def enum_variable():
 
+    class MyEnum(Enum):
```

Comment:
I understand we have a naming conflict with the `pytest` API. However, these new names are not satisfactory.
What about `Stub` or `Fixture`? 🙂